installer/ldap/install.sh

```diff
@@ -16,8 +16,9 @@
 #
 
 . ../utils/messages.sh
-cd ../../tests/deployment/ldap
+cd ../../helm-deployment/ldap-subchart/
 header "Installing LDAP"
-helm install --name imm-ldap -f values.yaml stable/openldap
+#helm install --name imm-ldap -f values.yaml stable/openldap
```

Comment:
?

Reply:
It was moved to ./deployment.sh in helm-deployment/ldap-subchart